actions/wit/README.md

```diff
@@ -0,0 +1,22 @@
+# Wit GitHub Action
+
+This [GitHub Action](https://github.com/features/actions) will run any arbitrary
+Wit command. Since it is packaged as a Docker-based command, it has all of its
+dependencies included and can be used to initialize and run commands on any Wit
+workspace.
+
+See [action.yml](./action.yml) for a detailed list of input parameters.
+
+## Example Usage
+
+```yaml
+- name: 'Wit Init'
+  uses: sifive/wit/actions/wit@v0.11.2
+  with:
+    command: init
+    arguments: |
+      workspace_dir
+      -a ./local-packages/api-scala-sifive
```

Comment:
The `local-packages` thing is a product of where you tell the Github Actions CI to put the repo right?

Reply:
Yeah, maybe I should reword this to `-a ./path/to/local/git/repo` to make it clearer that `local-packages` is not a special name.